src/sentry/integrations/bitbucket/integration.py

```diff
@@ -56,6 +58,21 @@ def get_repositories(self):
             )
         return data
 
+    def get_unmigratable_repositories(self):
```

Comment:
There's no need to change this. I almost feel like you could have added the functionality to the base class. As this method for VSTS is the same with the exception of the provider name. And surely somewhere there's a variable that has the name stored. That's just a nit though looks fine to me.

Reply:
Yea, the only difference is the `accessible_repos` bit (one of them needs to match on name – GitHub I think). Seemed less surprising to just leave the little bit of logic in each Integration.